src/views/Article/Index.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
@leksyib why are we disabling so much eslint rules?

Reply:
@9jaswag This rule is looking for a key event prop that we're not ready to pass, and we do not want to pass an empty function.